product-library.adoc

```diff
@@ -21,40 +21,83 @@ The Product Library API implements the following scopes:
 `POST /organizations/self/products`
 
 https://products.izettle.com/swagger/#!/products/createProduct
+
+*Request payload*
+
 ```json
 {
-  "uuid": "d62f7bb0-2728-11e6-85b5-dd108c223139",
-  "categories": ["shoes","clothing"],
-  "name": "ProductName",
-  "description": "Description of product",
-  "imageLookupKeys": [
-    "x0yH8KnREeequIvGpnO8Qw.jpg"
-  ],
-  "presentation" : {
-    "imageUrl" : "https://image.izettle.com/productimage/o/x0yH8KnREeequIvGpnO8Qw.jpg",
-    "backgroundColor" : "#5f5f5f",
-    "textColor" : "#7c7c7c"
-  },
-  "variants": [
-    {
-      "uuid": "d635e450-2728-11e6-a84e-d3d803b3d694",
-      "name": "VariantName",
-      "description": "Description of variant",
-      "sku": "SKU",
-      "barcode": "Barcode",
-      "price": {
-        "amount": 10,
-        "currencyId": "SEK"
-      },
-      "costPrice": {
-        "amount": 5,
-        "currencyId": "SEK"
-      }
-    }
-  ],
-  "externalReference": "externalReference",
-  "unitName": "Kg",
-  "vatPercentage": "10"
+	"uuid": "89f9d200-fb8a-11ea-adc1-0242ac120002",
+	"name": "Jeans",
+	"description": "Description of product.",
+    "imageLookupKeys": [
```

Comment:
@pypl-simon I can't remember if imageLookupKeys or presentation.imageUrl is preferred?

Reply:
`presentation.imageUrl` is the preferred one. `imageLookupKeys` is deprecated.